### PR TITLE
 Fixed docs for isInvalidEnrollmentReason

### PR DESCRIPTION
**Pull request by TrustHenry**
Modified the annotation of the isInvalidEnrollmentReason function to its meaning.

### CODE REVIEW

**Review by linked0 — APPROVED** [review]
Good change!
Removing 'Enter' in the description could be better.

**Comment by AndrejMitrovic** [comment]
The commit message is too long. Please rename it to something like `Fixed docs for isInvalidEnrollmentReason`

File: source/agora/consensus/Validation.d [hunk 1]
```diff
@@ -1002,7 +1002,7 @@ public string isInvalidReason (const ref Block block, in ulong prev_height,
         - The block height must be at least unlock_height or greater of the UTXO.
 
     Params:
-        block_height = the height of the direct ancestor of this block
+        block_height = Enter the height of the block to verified
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Validation.d]: `the height at which the enrollment wants to be added to the block`

**Review by AndrejMitrovic — CHANGES_REQUESTED** [review]
requesting some changes

File: source/agora/consensus/Validation.d (hunk 1 above)
Comment (TrustHenry) [reply source/agora/consensus/Validation.d]: applied the suggestion.

**Review by AndrejMitrovic — APPROVED** [review]

**Pull request merged**